RevenueCatUI/CustomerCenter/Views/WrongPlatformView.swift

```diff
@@ -92,15 +83,30 @@ struct WrongPlatformView: View {
                     }
                 }
             }
-            if let url = supportURL {
+
+            if let url = supportInformation?.supportURL(localization: localization) {
                 Section {
                     AsyncButton {
                         openURL(url)
                     } label: {
                         Text(localization[.contactSupport])
                     }
                 }
+            } else if RuntimeUtils.isSimulator {
+                AsyncButton { } label: {
+                    VStack(alignment: .leading) {
+                        Text(localization[.contactSupport])
+                            .frame(alignment: .leading)
+
+                        Text("Not available on simulator")
+                            .frame(alignment: .leading)
+                            .font(.caption)
+                            .foregroundStyle(.secondary)
```

Comment:
I am not sure if it's worth localizing this? What do you think @RevenueCat/coresdk ? I don't really want this to be shown in the customer center dashboard

Reply:
Yeah not worth it, it probably won't ever be seen by final users